ToDoApp/Backend/src/resolvers/user/userResolver.js

```diff
@@ -1,56 +1,105 @@
-const { find }= require('lodash');
-const { generateIntID }= require("../../../utils.js");
-const { CONFIG }= require("../../config/config");
+const {
+    find
+} = require('lodash');
+const {
+    generateUUID
+} = require("../../../utils.js");
+const {
+    CONFIG
+} = require("../../config/config");
 const bcrypt = require('bcryptjs')
 const jwt = require('jsonwebtoken')
+const {
+    neo4jgraphql
```

Comment:
Is this import used? I searched for `neo4jgraphql` and couldn't find where you use it.